shared/templates/static/bash/configure_firewalld_ports.sh

```diff
@@ -0,0 +1,11 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
+# reboot = false
+# complexity = low
+# strategy = configure
+# disruption = low
+
+. /usr/share/scap-security-guide/remediation_functions
+
+package_command install firewalld
```

Comment:
I know it was already there, but I wonder. I thought that there is a separate rule checking if package firewalld is installed? There could be also a separate remediation that just installs the package.

Reply:
There is separate Rule for firewalld installed with its own fix, but it is not selected anywhere.
Installing the package we are configuring during remediation is the way we are dealing with dependency on `package_foo_installed` Rules.
Not ideal, but this works around need of XCCDF requires.